# Publishes arrays of values obtained by the controller 
# to tune it with help of rosrun plotjuggler PlotJuggler
float32[] error_p
float32[] error_vel
float32[] integral_p
float32[] integral_vel
float32[] des_p
float32[] real_p
